tutorials/getting_started/getting_started.md

```diff
@@ -0,0 +1,181 @@
+It consists of a word embedding layer followed by a LSTM.
+
+Our dataset will be a subset of the [Brown Corpus](http://www.nltk.org/nltk_data/).
+In particular, the file [tutorials/getting_started/data/cr.train](https://github.com/allenai/allennlp/blob/master/tutorials/getting_started/data/cr.train)
+is the concatenation of the Brown files `cr01`, ..., `cr08` (that's the "humor" category)
+and [tutorials/getting_started/data/cr.dev](https://github.com/allenai/allennlp/blob/master/tutorials/getting_started/data/cr.test) is the file `cr09`.
+
+In AllenNLP we configure experiments using JSON files. Our experiment is defined in
+[tutorials/getting_started/simple_tagger.json](https://github.com/allenai/allennlp/blob/master/tutorials/getting_started/simple_tagger.json). You can peek at it
+if you want, we'll go through it in detail in the next tutorial.  Right at this instant
+you might care about the `trainer` section, which specifies how we want to train our model:
+
+```js
+  "trainer": {
+    "num_epochs": 20,
+    "serialization_prefix": "/tmp/tutorials/getting_started",
+    "cuda_device": -1
+  }
+```
+
+The `num_epochs` parameter just specifies how many passes through the dataset we want to train on.
+On a recent Macbook each epoch of this model on this dataset takes about 30 seconds, so 20 will take about 10 minutes.
+The `serialization` prefix is the path where the model's vocabulary and checkpointed weights will be saved.
+And if you have a GPU you can change `cuda_device` to 0 to use it.
+
+Change any of those if you want to, and then run
+
+```
+allennlp/run train tutorials/getting_started/simple_tagger.json
+```
+
+It will log all of the parameters it's using and then display the progress and results of each epoch:
+
+```
+2017-08-15 11:37:53,030 - INFO - allennlp.training.trainer - Epoch 5/20
+accuracy: 0.48, accuracy_top3: 0.63, loss: 2.19 ||: 100%|##########| 477/477 [00:25<00:00, 21.68it/s]
+accuracy: 0.50, accuracy_top3: 0.64, loss: 2.27 ||: 100%|##########| 50/50 [00:00<00:00, 53.72it/s]
+2017-08-15 11:38:19,609 - INFO - allennlp.training.trainer - Training accuracy : 0.477715    Validation accuracy : 0.501286
+2017-08-15 11:38:19,610 - INFO - allennlp.training.trainer - Training accuracy3 : 0.631720    Validation accuracy3 : 0.642796
+2017-08-15 11:38:19,610 - INFO - allennlp.training.trainer - Training loss : 2.187194    Validation loss : 2.265297
+2017-08-15 11:38:19,617 - INFO - allennlp.training.trainer - Best validation performance so far. Copying weights to /tmp/tutorials/getting_started/best.th'.
+```
+
+Here `accuracy` measures how often our model predicted the "correct" part of speech tag as most probable,
+while `accuracy3` measures how often the correct tag was one of the _three_ most probable.
+(`loss` measures [cross entropy](https://en.wikipedia.org/wiki/Cross_entropy), which you probably don't care about
```

Comment:
maybe state explicitly what it's a measure of.  "You probably don't care about the implementation details, but cross entropy measures whether the model is improving.  You should see that it's mostly decreasing during training."